math_hurdler.py

```diff
@@ -246,6 +249,9 @@ def evaluate_answer(answer):
             self.buttons[self.question.answer_index].set_color(Color.GREEN, False)
 
         while self.running:
+            #Processing Gtk Events
+            while Gtk.events_pending():
+                Gtk.main_iteration()
```

Comment:
Check self.running after processing GTK events, because one of the events may have been the stop button or accelerator.  See sugargame/test activity for how it is done.